MetaMask/metamask-mobile
# test: add Ethereum sign in e2e

## **Description**

This PR is within the scope of the Quality Quest.

We're including an e2e test case to check the sign in with Ethereum feature.

**Test steps:**
Given I am on the test dapp
When I tap on the Sign in with ethereum button
Then the Signature form should appear
When I tap cancel
Then the signature form should disappear
When I tap on the Sign in with ethereum button
And I tap Sign
Then the signature form should disappear
_And the signature result should not be empty_  ← This step is pending and will be done in the future.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**


### **Before**


### **After**


## **Pre-merge author checklist**

- [✓] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [✓] I've completed the PR template to the best of my ability
- [✓] I’ve included tests if applicable
- [✓] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [✓] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**
